greeting.rb

```diff
@@ -0,0 +1,42 @@
+class Greeting
+  def greet(*name)
+    if name.any?
```

Comment:
You can flip this `if` statement around by "returning early" which would reduce the nesting in the code. This is also known as a "guard clause": https://stackoverflow.com/a/37880756/998598